graphjet-core/src/test/java/com/twitter/graphjet/algorithms/counting/TopSecondDegreeByCountTest.java

```diff
@@ -159,7 +164,7 @@ public void testTopSecondDegreeByCountWithSocialProofTypeUnionsWithSmallGraph()
       new RequestedSetFilter(new NullStatsReceiver())
     ));
 
-    TopSecondDegreeByCountRequest topSecondDegreeByCountRequest = new TopSecondDegreeByCountRequest(
+    TopSecondDegreeByCountRequestForTweet topSecondDegreeByCountRequestForTweet = new TopSecondDegreeByCountRequestForTweet(
```

Comment:
Yeah, definitely see your point. On the other hand reviews may get filled up with line length, import ordering and spacing issues, and that too with imperfect recall. Happy to defer to others' sensibilities though